components/membership/registration-impl/src/main/kotlin/net/corda/membership/impl/registration/dynamic/member/DynamicMemberRegistrationService.kt

```
@@ -452,7 +453,7 @@ class DynamicMemberRegistrationService @Activate constructor(
             tenantId: String,
             expectedCategory: String,
         ): List<CryptoSigningKey> =
-            cryptoOpsClient.lookup(tenantId, keyIds).also { keys ->
+            cryptoOpsClient.lookupKeysByIds(tenantId, keyIds.map { ShortHash.of(it) }).also { keys ->
```

We need to throw the correct exception if the `keyId` is not a valid short hash (`InvalidMembershipRegistrationException`?)